tests/test_benchmarks/test_baker2022.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+from pytest import approx
+
+from brainio.assemblies import BehavioralAssembly
+from brainscore import benchmark_pool
+from brainscore.benchmarks.baker2022 import DATASETS
+from tests.test_benchmarks import PrecomputedFeatures
+
+@pytest.mark.private_access
+class TestBaker2022:
+
+    # ensure normal and inverted datasets are there
+    def test_count(self):
+        assert len(DATASETS) == 2
+
+    # ensure the three benchmarks themselves are there
+    @pytest.mark.parametrize('benchmark', [
+        'Baker2022-inverted_accuracy_delta',
+        'Baker2022-accuracy_delta_fragmented',
+        'Baker2022-inverted_accuracy_delta'
+    ])
+    def test_in_pool(self, benchmark):
+        assert benchmark in benchmark_pool
+
+    # Test expected ceiling
+    @pytest.mark.parametrize('benchmark, expected_ceiling', [
+        ('Baker2022-accuracy_delta_frankenstein', 0.8498),
+        ('Baker2022-accuracy_delta_fragmented', 0.9385),
+        ('Baker2022-inverted_accuracy_delta', 0.6538),
+    ])
+    def test_benchmark_ceiling(self, benchmark, expected_ceiling):
+        benchmark = benchmark_pool[benchmark]
+        assembly = benchmark._assembly
+        if "inverted" in benchmark.identifier:
+            inverted_assembly = assembly[assembly["orientation"] == "inverted"]
+            ceiling = benchmark._ceiling(inverted_assembly)
+        else:
+            ceiling = benchmark._ceiling(assembly)
+        assert ceiling == approx(expected_ceiling, abs=0.001)
+
+    # Test raw scores
+    @pytest.mark.parametrize('benchmark, model, expected_raw_score', [
+        ('Baker2022-accuracy_delta_frankenstein', 'resnet-50-pytorch', approx(0.2847, abs=0.0001)),
+        ('Baker2022-accuracy_delta_fragmented', 'resnet-50-pytorch', approx(0.8452, abs=0.0001)),
+        ('Baker2022-inverted_accuracy_delta', 'resnet-50-pytorch', approx(0.0, abs=0.0001)),
+        ('Baker2022-accuracy_delta_frankenstein', 'resnet50-SIN', approx(0.6823, abs=0.0001)),
+        ('Baker2022-accuracy_delta_fragmented', 'resnet50-SIN', approx(0.9100, abs=0.0001)),
+        ('Baker2022-inverted_accuracy_delta', 'resnet50-SIN', approx(0.7050, abs=0.0001)),
+
+    ])
+    def test_model_raw_score(self, benchmark, model, expected_raw_score):
+
+        # load features
+        precomputed_features = Path(__file__).parent / f'{model}-{benchmark}.nc'
+        benchmark = benchmark_pool[benchmark]
+        precomputed_features = BehavioralAssembly.from_files(file_path=precomputed_features)
+        precomputed_features = PrecomputedFeatures(precomputed_features,
+                                                   visual_degrees=8.8,  # doesn't matter, features are already computed
+                                                   )
+        score = benchmark(precomputed_features)
+        raw_score = score.raw
+        assert raw_score[0] == expected_raw_score
```

Comment:
```suggestion
        assert raw_score.sel(aggregation='center') == expected_raw_score
```
Do not use positional indexing when you can avoid it!